api/src/db/DynamoBusinessesDataClient.ts

```diff
@@ -64,10 +56,10 @@ export const DynamoBusinessesDataClient = (
           return [];
         }
         const businesses = result.Items.map((item) => unmarshall(item));
-        return Promise.all(businesses.map(async (item) => await doMigration(item.data)));
+        return Promise.all(businesses.map(async (item) => await item.data));
       })
-      .catch((error) => {
-        console.log(error);
+      .catch((error: Error) => {
+        logger.LogError(`Error finding business by index: ${indexName} with error: ${error.message}`);
         throw new Error("Error retrieving items");
       });
   };
```

Comment:
[dust] It might also be really helpful to have a function `findAllByBusinessName` given there's a strong likelihood we have multiple businesses with the same name.

Reply:
Added.